packages/node_modules/@ciscospark/redux-module-conversation/src/reducer.js

```diff
@@ -11,7 +11,7 @@ import {
   UPDATE_MERCURY_STATE
 } from './actions';
 
-export const initialState = {
+export const initialState = new Map({
   activities: new OrderedMap(),
   id: null,
   lastAcknowledgedActivityId: null,
```

Comment:
Yea, that's probably a good idea. Didn't want to do too much here as a "bug fix". I have that set up in the "spaces" I've been putting together.